docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-alerts-health-check.md

```diff
@@ -11,447 +11,184 @@ menu:
 type: docs
 ---
 
-YugabyteDB Anywhere can check universes for issues that may affect deployment. Should problems arise, YugabyteDB Anywhere can automatically issue alert notifications
+YugabyteDB Anywhere has default, preconfigured alerts, both at the YugabyteDB Anywhere and universe level. Universe alerts can be configured globally for all universes, or per specific universe. In addition to the default alerts, you can configure your own alerts based on a specific condition on any available metric.
```

Comment:
on any available alert policy template. 

We can't configure alerts based on any metric right now.